app-ios/Sources/Model/TimetableTimeGroupItems.swift

```diff
@@ -0,0 +1,19 @@
+import appioscombined
+
+public struct TimetableTimeGroupItems: Identifiable, Equatable, Hashable {
+    public var id: String {
+        items.first?.timetableItem.id.value ?? UUID().uuidString
+    }
+
+    public var startsAt: Date
+    public var endsAt: Date
```

Comment:
How about use `Duration` defined in `TimetableListView.swift`?
I think it should be useful!

Reply:
I'll try to refactor this 👍

Reply:
https://github.com/DroidKaigi/conference-app-2022/pull/402/commits/9e0e3354d1f9553f77c6a0eb8411538f1e5d665a

I refactored it.
I use `Duration` only in `init`.